### PR TITLE
Acronym

### PR DESCRIPTION
**Pull request by annaoliinyk**
https://exercism.org/tracks/python/exercises/acronym

Instructions
Convert a phrase to its acronym.

Techies love their TLA (Three Letter Acronyms)!

Help generate some jargon by writing a program that converts a long name like Portable Network Graphics to its acronym (PNG).

Punctuation is handled as follows: hyphens are word separators (like whitespace); all other punctuation can be removed from the input.

### CODE REVIEW

File: acronym/acronym_task.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+def abbreviate(words):
+    result = ""
+    words_as_list = split_phrase_into_list_words(words)
+    for word in words_as_list:
+        result += word[0].upper()
+    return result
+
+
+def split_phrase_into_list_words(phrase):
```
Comment (AnaKovaGH) [inline acronym/acronym_task.py]: Name is too long

File: acronym/acronym_task.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    phrase = phrase.replace("-", " ")
+    phrase = phrase.replace("_", " ")
+    phrase = phrase.replace("   ", " ")
+    phrase = phrase.replace("  ", " ")
```
Comment (AnaKovaGH) [inline acronym/acronym_task.py]: It is not effective. You're iterating phrase 4 times from the beginning till the end.

**Review by AnaKovaGH — CHANGES_REQUESTED** [review]
Please change what was requested

**Comment by annaoliinyk** [comment]
Fixed

**Review by AnaKovaGH — APPROVED** [review]

**Pull request merged**